### DISCUSSION TITLE
fa-solid-900.ttf not found on Lambda function

### DISCUSSION THREAD

**Discussion by gopjak36** [aminalaee/sqladmin#687] [Q&A] [answered]
Hi community. I have deployed sqladmin with lambda function, but I am getting a 404 Not Found error only for the 'admin/panel/statics/webfonts/fa-solid-900.ttf' URL. 
<img width="668" alt="Screenshot 2024-01-02 at 17 06 40" src="https://github.com/aminalaee/sqladmin/assets/24796728/3627d728-b834-4a55-acba-585421a50308">

It's uncomfortable not to have icons. All other requests are working well. Could you help me figure it out?

**Comment by rusanpas** [comment, marked as answer]
You have to activate gzip middleware in Fastapi to force the gateway API to decode the b64.

```python
app.add_middleware(GZipMiddleware, minimum_size=1000)
```

I also suggest, although I think it is optional, adding the mimetype font/woff2 to the .woff2 extension. As well as, in binary media types of the gateway API add font/woff2.

```python
mimetypes.add_type('font/ttf', '.ttf')
mimetypes.add_type('font/woff2', '.woff2')
```

It was hard for me to come up with this solution, but it works.
